src/QMCDrivers/WFOpt/QMCFixedSampleLinearOptimizeBatched.cpp

```diff
@@ -89,7 +89,14 @@ QMCFixedSampleLinearOptimizeBatched::QMCFixedSampleLinearOptimizeBatched(
       generate_samples_timer_(
           *timer_manager.createTimer("QMCLinearOptimizeBatched::GenerateSamples", timer_level_medium)),
       initialize_timer_(*timer_manager.createTimer("QMCLinearOptimizeBatched::Initialize", timer_level_medium)),
+      derivative_timer_(
+          *timer_manager.createTimer("QMCLinearOptimizeBatched::optTarget->checkConfigurations()", timer_level_medium)),
+      buildmat_timer_(
+          *timer_manager.createTimer("QMCLinearOptimizeBatched::optTarget->fillOverlapHamiltonianMatrices()",
+                                     timer_level_medium)),
```

Comment:
These timers already exist in QMCCostFunctionBatched.

Reply:
Removed.